src/Bicep.Core/Parser/TokenType.cs

```diff
@@ -5,6 +5,7 @@ namespace Bicep.Core.Parser
     public enum TokenType
     {
         Unrecognized,
+        Missing,
```

Comment:
I'm using SkippedSyntaxTrivia for missing nodes as well, which makes it have 0 length. Without something in the list of elements, I can't figure out the position. A zero length token can help me do that and avoid an extra constructor, but I think I will remove that regardless.

Reply:
Makes sense! How does this play with whitespace BTW? IIRC whitespace is owned by the preceding token, so the `Missing` token appears immediately before the next token?

Reply:
It's only inserted by the parser in one place. I might actually get rid of it because it didn't get me as much as I thought it would.

Reply:
Changing to use `SkippedTriviaSyntax` removed the need for the `Missing` token type, so I removed it.